src/com/sap/piper/PiperGoUtils.groovy

```diff
@@ -47,8 +47,10 @@ class PiperGoUtils implements Serializable {
                 }
             }
 
-            def fallbackUrl = 'https://github.com/SAP/jenkins-library/releases/latest/download/piper_master'
-            def piperBinUrl = (version == 'master') ? fallbackUrl : "https://github.com/SAP/jenkins-library/releases/download/${version}/piper"
+            // def fallbackUrl = 'https://github.com/SAP/jenkins-library/releases/latest/download/piper_master'
+            def fallbackUrl = 'https://github.com/chengshifan/jenkins-library/releases/latest/download/piper_master'
```

Comment:
Please revert these changes.

Reply:
OK. I changed it for testing and now it is reverted